src/syngen/ml/strategies/strategies.py

```diff
@@ -128,20 +128,25 @@ def run(self, **kwargs):
         Launch the training process
         """
         try:
+            table = kwargs["table_name"]
+            MlflowTracker().start_run(
+                run_name=f"{table}-PREPROCESS",
+                tags={"table_name": table, "process": "preprocess"},
+            )
             self.set_config(
                 source=kwargs["source"],
                 epochs=kwargs["epochs"],
                 drop_null=kwargs["drop_null"],
                 row_limit=kwargs["row_limit"],
-                table_name=kwargs["table_name"],
+                table_name=table,
                 metadata_path=kwargs["metadata_path"],
                 print_report=kwargs["print_report"],
                 batch_size=kwargs["batch_size"],
             )
-            MlflowTracker().log_params(self.config.to_dict())
```

Comment:
Don't we need to log parameters any more?

Reply:
These parameters like "epochs", "print_report", "batch_size", "row_limit" and etc. are added to the section "parameters" to bring more information about parameters set in the particular run.